src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.tsx

```diff
@@ -13,10 +14,11 @@ interface AssetManagementFormProps {
   formAction: AssetManagementActions;
   onConnectToWallet: () => void;
   onSetFormAction: (nextFormAction: AssetManagementActions) => void;
-  onTransferHolder?: (nextHolder: string) => void;
+  onTransferHolder: (nextHolder: string) => void;
   onEndorseBeneficiary?: (nextBeneficiary: string) => void; // Assuming holder is default to current holder
   surrenderingState: string;
   onSurrender: () => void;
+  holderState: string;
```

Comment:
```suggestion
  holderTransferringState: string;
```

if not it might be confused with the other actions on resolving `holder` address itself.

Reply:
got it!